### PR TITLE
fix: issue where score would update when hovering over levels during editing

### PR DESCRIPTION
**Pull request by MitchelPaulin**
https://rally1.rallydev.com/#/57732444928ud/custom/373260458992?detail=%2Fdefect%2F605245772387

### CODE REVIEW

File: d2l-rubric-editable-score.js [hunk 1]
```diff
@@ -391,12 +392,12 @@ Polymer({
...
-			if (!this._isStaticView() && !this._isEditingScore) {
+			if (!this._isStaticView() && !this.isEditingScore) {
 				this.overriddenStyling = this.scoreOverridden;
 			}
 		} else {
 			this.scoreOverridden = assessmentEntity.hasAction('clear-total-score-override');
-			if (!this._isStaticView() && !this._isEditingScore) {
+			if (!this._isStaticView() && !this.isEditingScore) {
 				this.overriddenStyling = this.scoreOverridden;
 			}
```
Comment (semibran) [inline d2l-rubric-editable-score.js:402]: Lol are we really running this same conditional two separate times 💀

File: d2l-rubric-criterion-mobile.js [hunk 2]
```diff
@@ -259,6 +270,7 @@ Polymer({
 
 	observers: [
 		'_onEntityChanged(entity)',
+		'_getCurrentScore(_selected, _hovered, _focused, _levelEntities)'
```
Comment (semibran) [inline d2l-rubric-criterion-mobile.js:273]:
```suggestion
		'_updateCurrentScore(_selected, _hovered, _focused, _levelEntities)'
```

File: d2l-rubric-criterion-mobile.js [hunk 3]
```diff
@@ -303,13 +315,18 @@ Polymer({
 
 	_getCurrentScore: function(selected, hovered, focused, levels) {
```
Comment (semibran) [inline d2l-rubric-criterion-mobile.js:316]:
```suggestion
	_updateCurrentScore: function(selected, hovered, focused, levels) {
```

File: d2l-rubric-editable-score.js (hunk 1 above)
Comment (MitchelPaulin) [reply d2l-rubric-editable-score.js:402]: Lol, didn't even notice this

**Review by semibran — APPROVED** [review]

**Review by devpow112 — APPROVED** [review]

**Pull request merged**